KBaseSearchEngine.spec

```diff
@@ -101,16 +101,15 @@ module KBaseSearchEngine {
         returns (SearchTypesOutput) authentication required;
 
     /*
-      Rule for sorting found results. 'key_name', 'is_timestamp' and
-      'is_object_name' are alternative way of defining what property
-      if used for sorting. Default order is ascending (if 
-      'descending' field is not set).
+      Rule for sorting found results. 'key_name' is the keys that are sorted by. 
+      'is_timestamp' and 'is_workspace_id' are shortcuts.
+      Default order is ascending if 'ascending' field is not set).
     */
     typedef structure {
         boolean is_timestamp;
-        boolean is_object_name;
+        boolean is_workspace_id;
```

Comment:
This should be `is_access_group_id` or something.

And long term we should probably have three fields

key_name
is_object_key (e.g. do we stick `key.` in front of it or not)
ascending

Reply:
And we should define the standard (non-object) keys.

Reply:
I assume you took out is_object_name because it's broken?